jest/helpers.ts

```diff
@@ -142,18 +142,18 @@ function getExecaOptions(options: SpawnOptions) {
 function handleTestFailure(
   cmd: string,
   options: SpawnOptions,
-  result: {[key: string]: any},
+  result: execa.ExecaReturnBase<string>,
   args: string[] | undefined,
 ) {
-  if (!options.expectedFailure && result.code !== 0) {
+  if (!options.expectedFailure && result.exitCode !== 0) {
```

Comment:
This was spamming the logs with false positives on the CI E2E tests.

Reply:
I know that some time ago we hit the issue where commands were failing and `execa` where reporting status codes as `undefined`, hopefully this change will catch all situations when there's a failure. 

More details: https://github.com/react-native-community/cli/pull/2152